lint-spec.ts

```diff
@@ -113,6 +115,37 @@ function getRelativePath(filepath: string): string {
   return p.relative(process.cwd(), filepath);
 }
 
+async function lintNonHrxTestDir(
```

Comment:
Please add a brief JSDoc comment for this function, make sure it captures a note about using the `:ignore-for: lint-hrx` option for non UTF-8 files

Reply:
I added that